src/common/config.ts

```diff
@@ -47,12 +47,24 @@ config.EXCLUDED_NAMESPACES = loadExcludedNamespaces();
 config.WORKERS_COUNT = Number(config.WORKERS_COUNT) || 10;
 config.SKOPEO_COMPRESSION_LEVEL = Number(config.SKOPEO_COMPRESSION_LEVEL) || 6;
 
-// return Sysdig endpoint information
+// return Sysdig v1 endpoint information
 if (config.SYSDIG_ENDPOINT && config.SYSDIG_TOKEN) {
```

Comment:
I am going to reorder these if statements incase the customer still has the old secret